care/emr/reports/report_utils.py

```
@@ -0,0 +1,125 @@
+
+    report_upload.save(skip_internal_name=True)
+
+    try:
+        report_upload.files_manager.put_object(
+            report_upload, output_bytes, ContentType=mime_type
+        )
+        report_upload.upload_completed = True
+        report_upload.save()
+    except Exception as e:
+        report_upload.delete()
+        raise e
```

Ruff (0.14.8)

Use `raise` without specifying exception name

Remove exception name

(TRY201)